### ISSUE TITLE
Transparent technical blocks probably shouldn't be glass

### ISSUE THREAD

**Issue by ljfa-ag** [Chisel-Team/Chisel#41]
Transparent technical blocks don't drop anything when they're broken without silk touch. Their material is Material.glass even though stone and iron is not really glass :p

They are instances of BlockGlass, which can mess up other mods' expectations on them. For instance, with my glass shards mod, they will actually drop shards. Now I can easily special-case that on my side, but maybe they shouldn't be glass when the only thing they have common with it is the transparency.

**Comment by tterrag1098** [comment]
Agreed, they shouldn't be glass material. I didn't even realize they were.

**Comment by Drullkus** [comment]
Potentially related to #12
